js/mmoda.instrument.js

```diff
@@ -676,10 +676,11 @@ function panel_title(srcname, param) {
           publish_result_type = 'success';
           if (renku_publish_textStatus == 'error') {
             if (typeof serverResponse === 'object' && serverResponse.hasOwnProperty('error_message'))
-              serverResponse = serverResponse.error_message;
+              serverResponse = `\"${serverResponse.error_message}\" - we will work to address the issue.`;
             else
-              serverResponse = serverResponse;
-            publish_response_title = 'Error while publishing to the Renku repository: ';
+              serverResponse = "we will work to address the issue.";
+            publish_response_title = 'An internal error occured while publishing to Renku:';
```

Comment:
I suggest shorter

```suggestion
            publish_response_title = 'Could not publish to Renku:';                        
```

Also, could we put the statuspage.io links somewhere?

Reply:
I will try to fit them...my concern is just about the space, since I wanted to fit the entire message on one line

Reply:
They would be urls, so they could even fit in icons?
Text might be further shortened. Yes, space is a concern.

Reply:
Much better like this

![image](https://user-images.githubusercontent.com/3757833/208453363-932da33a-3be8-4e07-a673-e44d21c1d477.png)